onboarding/guide-to-getting-started.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Guide to getting started
+description: A list of things to complete when you start
+---
+
+## Your first day
+
+_Goal: I’ve met my manager and some members of the engineering team (over lunch at Tataki). I’ve logged into slack, google (artsymail), 1Password, and know how to use the coffee machine._
+
+## Your first week
+
+_Goal: I can ssh into gravity (staging and production). I can PR to a repo in Artsy’s Github org. I know what the engineering teams are, what their main focuses are, and who leads them. I know my way around the Artsy site._
+
+### First week checklist
```

Comment:
It would be nice if we could give them a place to copy/paste this checklist to; i just confirmed you can't do it to yourself in a slack message.

Reply:
Not to go overly technical, but we could have Peril do this (listen for a new user being added to the Engineering group on GitHub, if so, schedule a slack message to them )

Reply:
I _think_ that maybe @erikdstock is referring to having it in a place where they can actually tick off their progress through these steps.

How about a GH issue? It will allow you to actually tick the boxes and save that state.

Reply:
Oh, that's a really nice idea!

Reply:
@alloy that sounds like a great idea. Will add it as an issue template!